securesystemslib/signer/_gcp_signer.py

```diff
@@ -71,6 +84,79 @@ def from_priv_key_uri(
 
         return cls(uri.path, public_key)
 
+    @classmethod
+    def import_(cls, gcp_keyid: str) -> Tuple[str, Key]:
+        """Load key and signer details from KMS
+
+        Returns the private key uri and the public key. This method should only
+        be called once per key: the uri and Key should be stored for later use.
+        """
+        if GCP_IMPORT_ERROR:
+            raise exceptions.UnsupportedLibraryError(GCP_IMPORT_ERROR)
+
+        client = kms.KeyManagementServiceClient()
+        request = {"name": gcp_keyid}
+        kms_pubkey = client.get_public_key(request)
+        try:
+            keytype, scheme = cls._get_keytype_and_scheme(kms_pubkey.algorithm)
+        except KeyError as e:
+            raise exceptions.UnsupportedAlgorithmError(
+                f"{kms_pubkey.algorithm} is not a supported signing algorithm"
+            ) from e
+
+        keyval = {"public": kms_pubkey.pem}
+        keyid = _get_keyid(keytype, scheme, keyval)
+        public_key = SSlibKey(keyid, keytype, scheme, keyval)
+
+        return f"{cls.SCHEME}:{gcp_keyid}", public_key
+
+    @staticmethod
+    def _get_keytype_and_scheme(algorithm: int) -> Tuple[str, str]:
+        """Return keytype and scheme for the KMS algorithm enum"""
+        keytypes_and_schemes = {
+            CryptoKeyVersion.CryptoKeyVersionAlgorithm.EC_SIGN_P256_SHA256: (
+                "ecdsa-sha2-nistp256",
```

Comment:
Minor nit: https://github.com/secure-systems-lab/securesystemslib/issues/363 suggests to deprecate `ecdsa-sha2-nistp256` as keytype in favor of the more succinct `ecdsa`.

Reply:
oh, I must have gotten these mixed up